src/main/java/com/facebook/openwifirrm/CustomJettyServerFactory.java

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.openwifirrm;
+
+import java.util.concurrent.TimeUnit;
+
+import org.eclipse.jetty.server.Connector;
+import org.eclipse.jetty.server.ForwardedRequestCustomizer;
+import org.eclipse.jetty.server.HttpConfiguration;
+import org.eclipse.jetty.server.HttpConnectionFactory;
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.server.ServerConnector;
+import org.eclipse.jetty.util.thread.QueuedThreadPool;
+import org.eclipse.jetty.util.thread.ThreadPool;
+import spark.embeddedserver.jetty.JettyServerFactory;
+import spark.utils.Assert;
+
+/**
+ * Creates Jetty Server instances. Majority of the logic is taken
+ * from JettyServerFactory with port logic added.
```

Comment:
please be more specific about what you changed, i.e. that you are supporting two ports instead of 1 (and anything else?)